css/properties/overscroll-behavior.json

```diff
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "overscroll-behavior": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/overscroll-behavior",
+          "support": {
+            "webview_android": {
+              "version_added": "63"
+            },
+            "chrome": {
+              "version_added": "63"
+            },
+            "chrome_android": {
+              "version_added": "63"
+            },
+            "edge": {
+              "version_added": null
```

Comment:
false. We could maybe consider -ms-scroll-chaining as an alternative, but not even sure of this, so false sounds good for the moment.